frontend/src/components/models.ts

```diff
@@ -100,6 +100,21 @@ export const supportedChains: Array<Chain> = [
     iconUrls: ['/networks/optimism.svg'],
     logoURI: '/networks/optimism.svg',
   },
+  {
+    chainId: '0x64', // 100 as hex
+    chainName: 'Gnosis Chain',
+    nativeCurrency: {
+      address: NATIVE_TOKEN_ADDRESS,
+      name: 'xDAI',
+      symbol: 'xDAI',
+      decimals: 18,
+      logoURI: 'https://docs.gnosischain.com/img/tokens/xdai.png',
+    },
+    rpcUrls: ['https://rpc.ankr.com/gnosis'],
+    blockExplorerUrls: ['https://gnosisscan.io'],
+    iconUrls: ['https://docs.gnosischain.com/img/tokens/gno.png'],
+    logoURI: 'https://docs.gnosischain.com/img/tokens/gno.png',
```

Comment:
I think that might be the icon for the GNO token for staking, but we want the chain logo which I think is green in color? https://uploads-ssl.webflow.com/63692bf32544bee8b1836ea6/636a6e764bdb11a70341fab4_owl-forest.png

Reply:
You're absolutely right! Updated the logo here: https://github.com/ScopeLift/umbra-protocol/pull/536/commits/7c12ebbe61e9afed82647c1deafb4e0afaf1f22c